### ISSUE TITLE
Can't get psychopy3 to run

### ISSUE THREAD

**Issue by njvack** [psychopy/psychopy#1898]
I'm on OS X 10.13.6, and tried this on framework builds of python 3.6.5 and 2.7.15.

Yeah, I know it's not properly released yet, but I've done:

* Clone the repo, switch to psychopy3 branch
* `pip install -e .`

Tons of stuff installs.

Then, I can do

`psychopyApp.py`

and I get two little python rockets down in the Dock, but no window ever shows up. I get some probably-not-very-informative messages:

```
2018-07-20 15:36:29.322 Python[77129:2429847] ApplePersistenceIgnoreState: Existing state will not be touched. New state will be written to (null)
03:36:30 PM: Debug: Adding duplicate image handler for 'Windows bitmap file'
03:36:30 PM: Debug: Adding duplicate image handler for 'Windows bitmap file'
03:36:30 PM: Debug: Adding duplicate animation handler for '1' type
03:36:30 PM: Debug: Adding duplicate animation handler for '2' type
03:36:30 PM: Debug: Adding duplicate image handler for 'Windows bitmap file'
03:36:30 PM: Debug: Adding duplicate animation handler for '1' type
03:36:30 PM: Debug: Adding duplicate animation handler for '2' type
```

The directory `~/.psychopy3/pavlovia` is created and has a couple .json files in it.

Any guidance on getting this to go (even if it's just "wait for the release") would be super helpful.

Thanks for all your work on this project!

**Comment by peircej** [comment]
Yes, the uninformative error messages are an issue. :-/
I'm assuming you've got git installed? If not that would certainly be an issue.

But beyond that ew certainly need to get to a better error message. Sometimes what happens is that the error message crops up after the app takes control of stderr/stdout (passing them to the coder output window) but then swallows them if it crashes.

Could you edit ~/.psychopy3/userPrefs.cfg so that the line showOutput is set to False. Then *hopefully* we'll get a more informative error.

thanks

**Comment by hoechenberger** [comment]
I am experiencing the same issue on macOS, with both the Python 2 and 3 bundles.

**Comment by njvack** [comment]
FYI, I grabbed the beta release .dmg and it does the same thing, though instead of getting two little python rockets in my dock I get two psychopy icons.

Just tried the python 3 one but yeah.

**Comment by peircej** [comment]
Could it be the gatekeeper block? In system settings > Security and Privacy is it showing up as blocked?

**Comment by peircej** [comment]
Basically, I'm at a bit of a loss to know what the problem could be - both bundles work on my machines and I can't think what the difference is

**Comment by peircej** [comment]
Delete the appData.cfg you created though. That file _shouldn't_ contain a section called coder and might cause a new way to crash if it does!

**Comment by njvack** [comment]
Gatekeeper bothered me about an unidentified developer when I first ran the app, but I said it was cool. I don't know where I can see blocked apps in sysprefs -> security and privacy — I have tabs for General, FileVault, Firewall, and Privacy.

Anyhow I don't think that would be part of my challenge in getting it going from a source install 🤔 

I guess adding more logging of the form "I got here!" to my source build might help me see this through. Gotta find a few hours to devote to debugging though.

**Comment by njvack** [comment]
Okay, deleted appData.cfg; no change. Also tried running with -b, -c, --no-splash, and --firstrun options.

**Comment by njvack** [comment]
tl;dr: Something seems to be wrong with the initial startup wizard.

I've run this with:

python -m trace --trace pychopyApp.py

and found that it was hanging in opening some kind of `shellCall()` call in `core.py` — a little more backtracing and it looks like that call is from `_psychopyApp.py`'s `_wizard` on line 387.

But hey, I thought: What if we run this muggle-style? And I replaced `_wizard()`'s contents with `pass` and now I have a working psychopy!

**Comment by njvack** [comment]
At first I thought "maybe it doesn't like pyenv" but that wouldn't explain why the release version were unhappy too, unless that version somehow dips into the pyenv environments too?

My python environments aren't quite as bad as [Randall Munroe's](https://xkcd.com/1987/) but it is kind of close.

**Comment by peircej** [comment]
OK, then we have a place to start. I can't actually recreate the problem - when I make my machine think it's doing a firstRun (triggering the wizard) the app still starts. That said, I think the startup wizard is causing more problems than solutions right now, so I think I'll just remove it entirely for now.

I might reintroduce a simpler alternative sometime in the future. The concept of checking the computer on the first run is a sensible one, but I think the execution of it was a bit over the top.

I'll remove that (and in the meantime I've fixed the no-git-available crashes) and build a new release today. Thanks for your help!
